### DISCUSSION TITLE
"Action Required: Fix Renovate Configuration" -- it's fixed, now what?

### DISCUSSION THREAD

**Discussion by pmcelhaney** [renovatebot/renovate#21075] [Request Help] [answered]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe your problem.

I copied a repository from a private Github account to a Github Enterprise account and then installed Renovate. It failed because the renovate.json points to shared config and I hadn't updated the location of the shared config. 

The issue should addressed now. How to I get the bot to try again? 

### Relevant debug logs.

<details><summary>Logs</summary>

```
{
  "configError": {
    "validationError": "Cannot find preset's package (github>REDACTED/renovate-config)",
    "message": "config-validation",
    "stack": "Error: config-validation\n    at fetchPreset (/home/ubuntu/renovateapp/node_modules/renovate/dist/config/presets/index.js:293:23)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Object.resolveConfigPresets (/home/ubuntu/renovateapp/node_modules/renovate/dist/config/presets/index.js:238:39)\n    at async mergeRenovateConfig (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/init/merge.js:219:61)\n    at async getRepoConfig (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/init/config.js:11:14)\n    at async initRepo (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/init/index.js:34:14)\n    at async Object.renovateRepository (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/index.js:45:18)\n    at async renovateRepository (/home/ubuntu/renovateapp/app/worker/index.js:260:26)\n    at async /home/ubuntu/renovateapp/app/worker/index.js:454:5"
  },
  "res": "created"
}
```

</details>

**Comment by rarkins** [comment, marked as answer]
It should run if it detects you made a change to your `renovate.json`. If you log into app.renovatebot.com you can see if a job did indeed run then. Worst case if you don't want to make further commits or they are not working then you can remove, save, then add back the repo in the installation and that should trigger another webhook.

**Reply by pmcelhaney** [reply]
Thanks. After an hour or so it ran again. I have couple of other much smaller, simpler repositories that ran after a few minutes. Guess I just need to be patient. :)
